steelthread-0/walkthrough.md

```diff
@@ -25,113 +28,229 @@ You should also take the opportunity to login your oc CLI as this user. You can
 
 # Setting up a Enmasse queue in your namespace 
 
+- From your namespace click the catalog button on the left hand nav
 - Provision a Enmasse service from the catalog (choose EnMasse standard) name it eval and choose not to bind at this time.
 - Once complete, in the Eval namespace you will find a EnMasse provisioned service.
-- Create a binding to this provisioned service checking each available option.
+- Now create a binding to this provisioned service checking each available option. This will create a secret in the format `enmasse-standard-xxxxx-credentials-yyyyy`
 - Collapse the provisioned service and click on the dashboard link which will show on the right hand side of the EnMasse provisioned service. This link will bring you to the EnMasse dashboard. 
 - When asked to login click the OpenShift button and login with the same user that provisioned the service. In an eval environment this will be ```evals@example.com```
 - Create a new address in the EnMasse console named ```work-queue/requests``` choose the ```queue``` option and the pooled plan.
-- Create a second address in the EnMasse console named ```work-queue/worker-updates``` choose the multicast option and the standard plan
+- Create a second address in the EnMasse console named ```work-queue/worker-updates``` choose the ```queue``` option and the Pooled Queue.
 
-# Setup Enmasse connection in Fuse Ignite
+# Deploy the CRUD REST HTTP booster
 
-Back in your ```evals``` project, You will see a binding and a binding secret as part of the Enmaase provisioned service. There is an option to view this secret. Click on view and click on reveal. Take note of the following values
+This is the HTTP endpoint that will revieve the requests from the Fuse Ignite 
+integreation we create and add new fruits to the inventory.
 
-- messagingHost
-- username
-- password
 
-open the Fuse Ignite route and login in to the Fuse Ignite console using the evals@example.com user 
+Load up the launcher application. This is at ```https://launcher-launcher.<ROUTER_SUFFIX> ``` 
```

Comment:
This should be ```http``` rather than ```https```

Reply:
Looks like launcher was manually configured for https in cloudservices cluster.
I think this should be https though, so is there an ansible installer change needed to do that?

Reply:
Doesn't appear to be configurable via the launcher template: https://raw.githubusercontent.com/fabric8-launcher/launcher-openshift-templates/dbf6291c876f881a6d8544e664c9260c44879d7e/openshift/launcher-template.yaml

```
- apiVersion: v1
  kind: Route
  metadata:
    name: launcher
  spec:
    to:
      kind: Service
      name: launcher-frontend
      weight: 100
    wildcardPolicy: None
  status: {}
```

Reply:
Reverting to http for now.
I'll follow up on a https option for the template